README.md

```diff
@@ -1,3 +1,74 @@
-# complex
+# complex #
 
 [![windows](https://github.com/BlueQuartzSoftware/complex/actions/workflows/windows.yml/badge.svg)](https://github.com/BlueQuartzSoftware/complex/actions/workflows/windows.yml) [![linux](https://github.com/BlueQuartzSoftware/complex/actions/workflows/linux.yml/badge.svg)](https://github.com/BlueQuartzSoftware/complex/actions/workflows/linux.yml) [![macos](https://github.com/BlueQuartzSoftware/complex/actions/workflows/macos.yml/badge.svg)](https://github.com/BlueQuartzSoftware/complex/actions/workflows/macos.yml) [![clang-format](https://github.com/BlueQuartzSoftware/complex/actions/workflows/format_push.yml/badge.svg)](https://github.com/BlueQuartzSoftware/complex/actions/workflows/format_push.yml)
+
+## Introduction ##
+
+`complex` is a rewrite of the [SIMPL](https://www.github.com/bluequartzsoftware/simpl) library upon which [DREAM3D](https://www.github.com/bluequartzsoftware/dream3d) is written. This library aims to be fully C++17 compliant, removes the need for Qt5 at the library level and brings more flexible data organization and grouping options.
+
+## Prerequisites ##
+
+In order to compile `complex` you will need a C++17 compiler suite installed on your computer.
+
++ Compiler
+  + Windows Visual Studio 2017 v141 toolset
+  + macOS 10.15 and Xcode 12.4 or higher
+  + Linux with GCC Version 9.0 or higher
+
+## Install vcpkg ##
+
+The `complex` project uses the [vcpkg](https://www.github.com/microsoft/vcpkg) to manage it's dependent libraries. If this is not already installed on your system then you will need to download and compile it.
+
+### Windows ###
+
+Clone the vcpkg repository into a location that it will be used from. inside your home directory or at `C:/vcpkg` is a reasonable spot. *DO NOT USE A PATH WITH SPACES IN ANY OF THE FOLDERS*.
+
+```(lang-console)
+cd C:/Users/[USERNAME]/Applications
+git clone ssh://git@github.com/microsoft/vcpkg
+```
+
+Use the Windows Explorer to open the vcpkg directory and double click the `bootstrap-vcpkg.bat` file. This will create the `vcpkg.exe` file. You will need to ensure that the path to the `vcpkg.exe` is on your `PATH` environment variable. There are several ways to do this in Windows, select the one that is right for your environment.
```

Comment:
The bootstrap script will run automatically by CMake so this step is unnecessary. And adding to PATH is not required either.

Reply:
I think some users may need to add it to their path so at least a mention/reminder of it might be useful.